README.md

```diff
@@ -539,12 +541,17 @@ Query parameters that accept multiple values **MUST** be pluralized.
 
 If any request receives a query parameter it does not understand, the response **MUST** be a `400 Bad Request`.
 
+All query parameters **MUST** be properly [url-encoded](https://en.wikipedia.org/wiki/Percent-encoding). If a query parameter value includes the comma (`,`) character, the comma **MUST** be double encoded. Note that for readability purposes, the examples throughout this document do not show encoded query strings.
```

Comment:
Wouldn't it make sense to deep link onto https://en.wikipedia.org/wiki/Percent-encoding#Current_standard which specifies to use UTF-8 form for non ascii characters as to also solve https://github.com/cloudfoundry/cc-api-v3-style-guide/issues/48 with this PR ?